app/gui2/shared/languageServer.ts

```diff
@@ -138,10 +138,12 @@ export class LanguageServer extends ObservableV2<Notifications & TransportEvents
   private initializationScheduled = false
   private shouldReconnect = true
   private retainCount = 1
+  debug: boolean
```

Comment:
This property does not seem to be used by LanguageServer class, it's just read in LanguageServerSession, so I would remove the property from here and add it there.

Reply:
Now it is being used.